TTD/docs/TTDLiveRecorder.h/interface-ILiveRecorder.md

```diff
@@ -30,9 +30,9 @@ may be used to ensure lifetime is handled correctly.
 
 These methods control the access and lifetime of the object that provides the interface.
 
-- [`AddRef`](https://learn.microsoft.com/en-us/windows/win32/api/unknwn/nf-unknwn-iunknown-addref) (IUnknown)
-- [`Release`](https://learn.microsoft.com/en-us/windows/win32/api/unknwn/nf-unknwn-iunknown-release) (IUnknown)
-- [`QueryInterface`](https://learn.microsoft.com/en-us/windows/win32/api/unknwn/nf-unknwn-iunknown-queryinterface(refiid_void)) (IUnknown)
+- [`AddRef`](https://learn.microsoft.com/windows/win32/api/unknwn/nf-unknwn-iunknown-addref) (IUnknown)
+- [`Release`](https://learn.microsoft.com/windows/win32/api/unknwn/nf-unknwn-iunknown-release) (IUnknown)
+- [`QueryInterface`](https://learn.microsoft.com/windows/win32/api/unknwn/nf-unknwn-iunknown-queryinterface(refiid_void)) (IUnknown)
 
```

Comment:
this one needs en-us

Reply:
No it doesn't? Just tried it, works fine.

Reply:
Note I removed "en-us" following @DOMARS instructions.

Reply:
Aha! The parenthesis are confusing CodeFlow. All others (VSCode and GitHub) seem to be able to handle it fine.

I'll escape it anyway.